src/lib/packager/product_patterns.rb

```diff
@@ -99,8 +99,6 @@ def dependencies(product)
         end
       end
 
-      log.debug "Product #{product} depependencies: #{product_dependencies}"
```

Comment:
You are smuggling in an undocumented change, and I have a better one to replace it :smile: